include/curl/curl.h

```diff
@@ -2051,7 +2052,8 @@ typedef enum {
   /* Head start in milliseconds to give happy eyeballs. */
   CURLOPT(CURLOPT_HAPPY_EYEBALLS_TIMEOUT_MS, CURLOPTTYPE_LONG, 271),
 
-  /* Function that will be called before a resolver request is made */
+  /* Function that will be called before a resolver request is made.
+   * Only supported by the c-ares DNS backend */
```

Comment:
`CURLOPT_RESOLVER_START_FUNCTION` is not c-ares specific!

Reply:
Will review.